addons/block_code/blocks/communication/is_in_group.tres

```diff
@@ -10,8 +17,11 @@ description = "Is this node in the group"
 category = "Communication | Groups"
 type = 3
 variant_type = 1
-display_template = "is in group {group: STRING}"
-code_template = "is_in_group({group})"
-defaults = {}
+display_template = "is in group {group: NIL}"
+code_template = "is_in_group(\\\"{group}\\\")"
+defaults = {
+"group": SubResource("Resource_d0v0d")
+}
```

Comment:
Ack, I completely missed this change! We shouldn't have needed to change group to a NIL :( I'll make a PR fixing this in a moment.

Reply:
To explain what's going on, _previously_ we needed to do that for options lists because the value was just reproduced verbatim, but now it can be treated as a string and auto-quoted just like any other string parameter. The old behaviour was used in places, and that can be achieved by using something like `{{param}}` in the code template to substitute just the raw value.

Reply:
What was the change that made that different? I've been basing my work on 41afa00261e17606c0754b8697940f65c79c0194 where several string types changed to `NIL`.

Reply:
Oh, I see what happened there! Some crossed wires :) I think I switched them to NIL to keep the previous behaviour of not being able to snap weird things to options lists, but then #219 happened around the same time so it was kind of moot, and the parameter type is indeed pretty irrelevant to anything. The only thing that's important is we don't need to do anything special to quote the strings here.

The thing that caused the behaviour change in that commit is where we remove the special case for `TYPE_OBJECT` in `raw_input_to_code_string` (seeing as option values are no longer `OptionData` objects).